Model/Cron.php

```diff
@@ -1521,6 +1521,8 @@ protected function _authorizePayment()
         if ($isFullAmountAuthorized) {
             $this->_setPrePaymentAuthorized();
             $this->_prepareInvoice();
+            //set authorized amount in sales_order_payment
+            $this->_order->getPayment()->setAmountAuthorized($this->orderAmount);
```

Comment:
This will only set the field once the full amount has been authorized. However when a partial payment is processed the field won't be updated. Can we update the else statement to update this field by adding the amount of this webhook to the existing amount authorized?

Reply:
I believe we need to update the amount only in case is full authorised similar to cc that is the reason why I added only after the amount is full authorised

Reply:
We're going to have a conflict with #1309 here though

Reply:
Good point @peterojo lets work on this after you merge the cron refactor

Reply:
I think this can be picked up again, but in a different file as Cron.php is being removed